code/parachain/frame/crowdloan-rewards/src/lib.rs

```diff
@@ -339,6 +341,36 @@ pub mod pallet {
 			Self::do_unlock(reward_accounts);
 			Ok(())
 		}
+
+		/// Deletes all accounts provided in the `deletions` vector. If the provided `Balance` does
+		/// not match the current balance of the account, the deletion will fail.
+		//
+		// README: Disabled to reduce the amount of auditing work. When enabling, also enable the
+		// tests.
+		//
+		// #[pallet::weight(<T as Config>::WeightInfo::populate(deletions.len() as _))]
+		// pub fn delete(
+		// 	origin: OriginFor<T>,
+		// 	deletions: Vec<(RemoteAccountOf<T>, BalanceOf<T>)>,
+		// ) -> DispatchResult {
+		// 	T::AdminOrigin::ensure_origin(origin)?;
+		// 	Self::do_delete(deletions.clone())?;
+		// 	Self::deposit_event(Event::RewardsDeleted { deletions });
+		// 	Ok(())
+		// }
+
+		/// Adds all accounts in the `additions` vector. Add may be called even if the pallet has
+		/// been initialized.
+		#[pallet::weight(<T as Config>::WeightInfo::populate(additions.len() as _))]
+		pub fn add(
+			origin: OriginFor<T>,
+			additions: Vec<(RemoteAccountOf<T>, RewardAmountOf<T>, VestingPeriodOf<T>)>,
```

Comment:
Could it be better to use a `BoundedVec` here?  Looks like we use `Vec` all the way through the pallet so maybe a subsequent thing to refactor.

Reply:
Technically it is bounded by the maximum size of a transaction.